sjb/config/test_cases/test_branch_openshift_ansible_logging.yml

```diff
@@ -19,33 +19,14 @@ extensions:
         fi
         hack/build-images.sh
     - type: "script"
-      title: "build an openshift-ansible release"
-      repository: "openshift-ansible"
-      script: |-
-        tito_tmp_dir="tito"
-        rm -rf "${tito_tmp_dir}"
-        mkdir -p "${tito_tmp_dir}"
-        tito tag --offline --accept-auto-changelog
-        tito build --output="${tito_tmp_dir}" --rpm --test --offline --quiet
-        createrepo "${tito_tmp_dir}/noarch"
-        cat << EOR > ./openshift-ansible-local-release.repo
-        [openshift-ansible-local-release]
-        baseurl = file://$( pwd )/${tito_tmp_dir}/noarch
-        gpgcheck = 0
-        name = OpenShift Ansible Release from Local Source
-        EOR
-        sudo cp ./openshift-ansible-local-release.repo /etc/yum.repos.d
-    - type: "script"
-      title: "install the openshift-ansible release"
-      repository: "openshift-ansible"
+      title: "install Ansible and other packages needed for OpenShift Ansible tasks"
+      repository: "origin-aggregated-logging"
       script: |-
-        last_tag="$( git describe --tags --abbrev=0 --exact-match HEAD )"
-        last_commit="$( git log -n 1 --pretty=%h )"
-        sudo yum install -y "atomic-openshift-utils${last_tag/openshift-ansible/}.git.0.${last_commit}.el7"
-        rpm -V "atomic-openshift-utils${last_tag/openshift-ansible/}.git.0.${last_commit}.el7"
+        sudo yum install -y ansible python2 python-six tar java-1.8.0-openjdk-headless httpd-tools \
```

Comment:
These sorts of tasks should be pushed into the base AMI: https://github.com/openshift/origin-ci-tool/blob/master/oct/ansible/oct/roles/dependencies/tasks/install_dependencies.yml#L5

`yum` tasks are notoriously slow and flaky.

Reply:
These packages were being installed as a "side effect" of doing the ansible package installation.  So this job will be doing `yum install` here anyway, no matter what.  I will note that almost all of them are already installed - I think just the java, httpd, and selinux packages are not already installed.

Reply:
Sure, in any case -- whether `yum` is resolving them or they're explicitly listed -- add them to the base AMI so we can no-op on them as much as possible, please

Reply:
So the base ami builder tracks the Requires and BuildRequires in the openshift-ansible.spec?

Reply:
No, it's just a static list, but we should aim to get all of our runtime RPM dependencies in there so we can lean up the jobs